content/02.introduction.md

```diff
@@ -18,7 +18,7 @@ This workflow provides a strong and clear advantage over receiving, processing a
 Through the combination of version control management and the network- and collaboration-based features, GitHub can broadly facilitate openly available source code alongside concomitant collaborative development [@doi:10.1371/journal.pcbi.1004947].
 
 Git is the version control system that enables all the collaborative tools available on GitHub.
-Although the understanding of basic concepts of Git (such as commit, push, pull, checkout; see [Box 1](#definitions)) is necessary, the GitHub web-based platform and its integrated development environments (such as the GitHub Desktop) allow users to manage their repositories without using more technical command-line sessions. 
+Although the understanding of basic concepts of Git (such as commit, push, pull, clone; see [Box 1](#definitions)) is necessary, the GitHub web-based platform and its integrated development environments (such as the GitHub Desktop) allow users to manage their repositories without using more technical command-line sessions. 
```

Comment:
"checkout" wasn't in Box 1, and I thought "clone" was a more important git skill for using GitHub anyways.  Could also revert and add a definition of "checkout"

Reply:
I agree - clone is probably more important to know than checkout